css/components/rating.css

```diff
@@ -0,0 +1,16 @@
+.rating-wrapper {
+  padding: 2rem 4rem;
+  border-radius: 4px;
+  box-shadow: 0 0 15px 10px rgba(0, 0, 0, 0.05);
```

Comment:
use variable

Reply:
Thanks Varun for the review, I have made the suggested changes. 😄